routes/api.js

```diff
@@ -288,13 +288,14 @@ var getMembersBySkills = async function (req, res, next) {
  * @apiUse OnNotFoundError
  */
 var addMember = function (req, res, next) {
+    console.log('req body', req.body);
```

Comment:
Do we need debug comments in code checked into master? I recommend removing these console logs before committing to master or using debug(). This will fire only when you enable debug at startup

Reply:
will clean this up.